prow/plugins/dco/dco.go

```diff
@@ -66,10 +67,27 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin does not support
-	// per-repo config
+	configInfo := map[string]string{}
+	for _, orgRepo := range enabledRepos {
+		parts := strings.Split(orgRepo, "/")
+		var opts *plugins.Dco
+		switch len(parts) {
+		case 1:
+			opts = config.DcoFor(parts[0], "")
+		case 2:
+			opts = config.DcoFor(parts[0], parts[1])
+		default:
```

Comment:
Just return an error? It doesnt make sense to have an empty string in `enabledRepos`

Reply:
Fixed